content/en/blog/_posts/2018-07-10-coredns-ga.md

```diff
@@ -0,0 +1,179 @@
+---
+layout: blog
+title: "CoreDNS GA for Kubernetes Cluster DNS"
+date: 2018-07-08
+---
+
+**Author**: John Belamaric (Infoblox)
+
+## Introduction
+
+In Kubernetes 1.11, [CoreDNS](https://coredns.io) has reached General Availability (GA) for DNS-based service discovery, as an alternative to the `kube-dns` addon. This means that CoreDNS will be offered as an option in upcoming versions of the various installation tools. In fact, the `kubeadm` team chose to make it the default option starting with Kubernetes 1.11.
```

Comment:
remove the " ` " around names like "kube-dns" and "kubeadm"? 
these are project names and not a pod name, a service name or command.

marking this as [1] but i didn't cover all places.